config/7.11.0/common.yaml

```diff
@@ -940,6 +940,24 @@ servers:
                       value: "[[.Kafka.MaxBlockMs]]"
                       #[[end]]
                     #[[end]]
+                    #[[if .KafkaJbpm]]
```

Comment:
It would be good to have there also some flag, like we have for Kafak integratino KIE_SERVER_KAFKA_EXT_ENABLED. We would need custom flag for KafkaJbpmEventEmitter (e.g. `KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ENABLED`) so then we could easily check in Kie Server image that a new configuration is needed and emitter jar needs to be moved to the correct place in ROOT.war

Reply:
We could set but in the jboss-kie-modules because in the operator if the user open the kafka extension we are considering enabled (if configured correctly) see https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh#L760

Reply:
ok